zest/releaser/pypi.py

```diff
@@ -423,30 +423,31 @@ def version_levels(self):
             return default
         return result
 
-    def tag_format(self):
-        """Return the string format for the tag name used in the release.
+    def tag_format(self, version):
+        """Format the tag that should be used in the release.
 
-        Override the default ``%(version)s`` in ~/.pypirc or setup.cfg using
+        Override the default ``{version}`` in ~/.pypirc or setup.cfg using
         a ``tag-format`` option::
 
             [zest.releaser]
-            tag-format = v%(version)s
+            tag-format = v{version}
 
-        Returns default of ``%(version)s`` when nothing has been configured.
```

Comment:
This line should stay, right?  (just change the % for { )

Your new line can also stay, but the original has its use, too.